### PR TITLE
Nissan: use cleaner Leaf brake pressed signal

### PR DESCRIPTION
**Pull request by TheKirkMiller**
Proposed fix for issue  #19631

OpenDBC and Panda require changes as well.

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
@TheKirkMiller that route doesn't have the rlogs uploaded. Can you post some routes that have this issue with rlogs uploaded?

**Comment by TheKirkMiller** [comment]
Sorry, check this route with partially updated code where it was triggered by thinking I was going to hit the road closed sign at 1165 seconds.

f49cbfe935911658|2021-10-31--16-00-58

File: selfdrive/car/nissan/carstate.py [hunk 1]
```diff
@@ -184,8 +184,8 @@ def get_can_parser(CP):
 
     elif CP.carFingerprint in [CAR.LEAF, CAR.LEAF_IC]:
       signals += [
-        ("BRAKE_PEDAL", "BRAKE_PEDAL", 0),
-
+        ("USER_BRAKE_PRESSED", "CRUISE_THROTTLE", 1),
+
```
Comment (adeebshihadeh) [inline selfdrive/car/nissan/carstate.py:188]:
```suggestion
        ("USER_BRAKE_PRESSED", "CRUISE_THROTTLE", 1),
```

**Comment by adeebshihadeh** [comment]
Thanks for tackling this @TheKirkMiller! Continued in leaf-brake since I wasn't able to push to your branch.

**Pull request closed without merging**